layers/meta-balena-beaglebone/recipes-core/images/balena-image.bbappend

```diff
@@ -1,4 +1,8 @@
 include balena-image.inc
 
+# This might be probably removed or reduced once the kernel gets compressed.
+IMAGE_ROOTFS_SIZE:beaglebone-ai64 = "458752"
+IMAGE_ROOTFS_MAXSIZE:beaglebone-ai64 = "1048576"
```

Comment:
Why override `IMAGE_ROOTFS_MAXSIZE`? The [default](https://github.com/balena-os/meta-balena/blob/f98f94b3181bfd112b2b88287ae29b134bb4c1ab/meta-balena-common/recipes-core/images/balena-image.bb#L17) in meta-balena is better.

Reply:
At the moment of creating the branch I could not get to build the image without increasing these values. I don't know if this changed in the meantime, but definitely worth checking.

Reply:
Increasing `IMAGE_ROOTFS_SIZE` is fine - it's just `IMAGE_ROOTFS_MAXSIZE` that's not needed.